src/App.js

```diff
@@ -64,6 +65,7 @@ const App = () => (
                   <Route path="/favourites" component={Favourites} />
                   <Route path="/my-bids" component={SearchResults} />
                   <Route path="/about" component={About} />
+                  <Route path="/demo" component={Demo} />
```

Comment:
Would need to remove before merging I think

Reply:
Right, will remove it once @Arachnid says so